relnotes/verify.feature.md

```diff
@@ -0,0 +1,4 @@
+* `ocean.core.Verify`
+
+  New module with a single function, `verify`, intended to serve as drop-in
+  replacement for `assert` to comply to Sociomantic assert/enforce policies.
```

Comment:
AFAIK out assert-vs-enforce guide is not available in public place anywhere. Would be nice to do that so it can be linked.